bindata/bootkube/scc-manifests/0000_20_kube-apiserver-operator_00_scc-hostnetwork-v2.yaml

```diff
@@ -0,0 +1,47 @@
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: true
+allowHostPID: false
+allowHostPorts: true
+allowPrivilegeEscalation: false
+allowPrivilegedContainer: false
+allowedCapabilities:
+- NET_BIND_SERVICE
+apiVersion: security.openshift.io/v1
+defaultAddCapabilities:
+fsGroup:
+  type: MustRunAs
+groups: []
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    kubernetes.io/description: hostnetwork allows using host networking and host ports
+      but still requires pods to be run with a UID and SELinux context that are allocated
+      to the namespace. On top of the legacy 'hostnetwork' SCC, it also requires to drop
+      ALL capabilities and does not allow privilege escalation binaries. It will also
+      default the seccomp profile to runtime/default if unset, otherwise this seccomp
```

Comment:
This defaulting behavior, it's different than the previous hostnetwork?  I'd like to move to less defaulting, not more.

Reply:
The defaulting is actually required for the PodSecurity restricted level. An empty value should mean the same as this defaulted one.